### DISCUSSION TITLE
四川大学（望江校区）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#6] [University]
## 大三大四
上下都是床，四人间，有空调，没独卫，有蚊子，有"广东"蟑螂，目前没见到老鼠
没有早自习，无晨跑，不过体测还是有的，没有跑步打卡

允许外卖，不能进学校，除非快递员从南门混进来

地铁出门就是

洗衣房，一桶五元价格不变

校园网：
免费的限时，略微限速
十块钱一个月可以破时长
二十一个月可以加速
三十一个月可以同时满足上两项

十二点断电所以十二点之后还是没校园网
可以办流量卡二十多一个月，但是你会被广告短信骚扰

食堂：
桃林面馆：北方人也可以点不辣的菜，价格合理，中规中矩，距离教学楼近，下课了过去吃饭比较方便
西门食堂：中规中矩。
北园食堂：冒菜味道不错但是对于非本地人可能太辣，也有卖单份的菜但是贵一点
活动中心食堂：黑心食堂

食堂土豆不削皮，土豆爱好者表示mmp

> cc by nc sa 3.0
> "131 的群"

**Comment by XuYang233** [comment]
现在校园网12点后有WiFi了，有线貌似还是没有。校园网仅在宿舍范围才限时100h-30M，只要选择了校园网套餐就是不限时
| 价格/元 | 终端数/台 | 带宽/M |
|----|-----|------|
| 10 | 1   | 30   |
| 20 | 1   | 60   |
| 30 | 1   | 90   |
| 30 | 2   | 30   |
| 40 | 2   | 60   |
| 50 | 2   | 90   |
| 50 | 3   | 30   |
| 60 | 3   | 60   |
| 70 | 3   | 90   |

但是我的建议是开学校的流量卡，100M不限时可以拿个路由器（望江与江安有些不同，有线必须用客户端才能登录，无线有网页认证，因此路由器需要每次自己登录或者用软路由），但最好自己去营业厅办理而不是学长学姐推销的，价格有29-40的不等，据我所知移动是名义上的29/月，但是续费需要100/年，有些坑

**Comment by a5sl** [comment]
有些宿舍是整楼翻新过了，六人间，是特殊的上床下桌（四上铺两下铺）。每个人桌子巨小，阳台也小，储物柜更是小。一个卫生间。
南门出去超多好吃的。地铁站在北门、西门，要是宿舍在校园中部走出去比较久。
